google-cloud-spanner/test/google/cloud/spanner/fields/initializer_test.rb

```diff
@@ -16,13 +16,13 @@
 
 describe Google::Cloud::Spanner::Fields, :initializer do
   it "creates with an array of fields" do
-    fields = Google::Cloud::Spanner::Fields.new [:INT64, :STRING, :BOOL]
+    fields = Google::Cloud::Spanner::Fields.new [:INT64, :STRING, :BOOL, :JSON]
```

Comment:
What is the motivation to add `:JSON` here?

Reply:
Just verify protobuf definition has JSON type available.